### PR TITLE
Google oauth2

### PR DESCRIPTION
**Pull request by bakaydmytro**
Google oauth2

to signup or login using a google account send get request to http://localhost:8000/api/sessions/google/redirect

### CODE REVIEW

File: src/controllers/authController.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        }
+
+        return sendResponse.sendSuccessResponse(res, 200, {
+            user: {
+                name: user.name,
+                id: user.id,
+                email: user.email,
+                accessToken: token
+            },
+            publicKey: authJwt.getPublicKey()
```
Comment (Jeffsingh) [inline src/controllers/authController.js]: Why are we sending the public key here?

Comment (Jeffsingh) [reply src/controllers/authController.js]: I don't think this should be needed...just the access token...

**Review by Jeffsingh — APPROVED** [review]
LGTM -- Just check the comments...

File: src/controllers/authController.js (hunk 1 above)
Comment (bakaydmytro) [reply src/controllers/authController.js]: ok, I'll change this

**Review by Jeffsingh — APPROVED** [review]

**Pull request merged**